dbpedia_Classical_musicians_by_nationality/package.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
I think you don't need an extra package.json for the scripts!.. Also :D version 1.0.0? Wow, that was a quick development cycle